README.md

```diff
@@ -1,44 +1,140 @@
+
+This is still a work in progress and there are definitely still many bugs. For example, the table columns for search results aren't sized correctly and when you condense the window width-wise while on a show page the text overlaps with the image. Furthermore, we have a lot of functionality in our extensions that we weren't able to complete in the 10 days alotted for this project. That is why we have announced on [our website](https://obituary-hub.herokuapp.com/resources) that this project will now be open source and we are inviting others in the Turing community and beyond to make contributions.
+
 ## Tech Stack
 
  - Ruby 5.1.7
  - Rails 5.2.4.3
  - PostgreSQL
- - Gems
-   * [Fast JSON API](https://github.com/Netflix/fast_jsonapi)(Thanks Netflix!)
-   * [Rack CORS Middleware](https://github.com/cyu/rack-cors)
-   * [Factory Bot](https://github.com/thoughtbot/factory_bot_rails)
-   * [Faker](https://github.com/faker-ruby/faker)
+ - Sendgrid
+ - JQuery
+ - Heroku
+ - TravisCI
+ - Sinatra
+ - Omniauth
 
 ## Schema
 
 ![Schema](/public/images/schema.png)
 
-## API
+## Design Strategy
+
+Since this was a school project, we tried our best to demonstrate our understanding of the 4 Pillars of Object Oriented Programming as well as ReSTful APIs and ReSTful routes. Evidence of these efforts can be found in the encapsulation of our microservice, our use of partials for rendering the different views, and our adherence to MVC. Within the Turing community, MVC means "skinny controllers" and "bulkier models"; therefore, we pushed all logic into models (as well as poros and facades).
+
+All our API endpoints (found below) are ReSTful, compliant with the JSON API spec through the use of serializers, and exposed under an api/v1 namespace. Versioning will come in handy as this project continues to expand; when we make changes to the API down the road, we will be able to distinguish those changes using a `v2` namespace.
+
+It is our understanding that the emphasis on ReSTful architecture at Turing is because this makes our app easier for both end users and developers to engage with. Following best practices isn't just a flex - it is an act of kindness.
+
+## Microservice <a name="microservice"></a>
+For this project, we used [The New York Times API](https://developer.nytimes.com/), and consumed it using a Sinatra microservice, which was built by [Jack Cullen](https://github.com/jpc20) and [Eric Larson](https://github.com/EricLarson2020).
+
+You can find the endpoints from our microservice listed in the README for that project [here](https://github.com/jpc20/obituary_microservice)
+
+### NYT Endpoints:
+
+All requests below are tacked onto the end of this root URL: [https://api.nytimes.com/svc/search/v2//articlesearch.json](https://api.nytimes.com/svc/search/v2//articlesearch.json)
```

Comment:
Are these 2 next lines a markdown trick too?

Reply:
the first one in the brackets shows and the second one in the parenthesis makes it a hyperlink. That's what I did with each person's name as well - it links to your github